api/components/com_users/src/Controller/UsersController.php

```diff
@@ -63,6 +63,11 @@ protected function preprocessSaveData(array $data): array
 			}
 		}
 
+		if ((isset($data['password'])) && ($this->task !== 'add'))
```

Comment:
```suggestion
		if (isset($data['password']) && $this->task !== 'add')
```
We do not need these inner brackets, operator precedence does the job right already.